local-volume/helm/provisioner/templates/provisioner-cluster-role-binding.yaml

```diff
@@ -37,3 +37,29 @@ roleRef:
   name: local-storage-provisioner-node-clusterrole
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think we want a Role (permissions in a specific namespace), rather than a ClusterRole (permissions across all namespaces)